.golangci.yaml

```
@@ -138,14 +138,44 @@ linters-settings:
     min-len: 3
     # minimal occurrences count to trigger, 3 by default
     min-occurrences: 8
-  # depguard:
-  #   list-type: blacklist
-  #   include-go-root: false
-  #   packages:
-  #     - github.com/sirupsen/logrus
-  #   packages-with-error-messages:
-  #     # specify an error message to output when a blacklisted package is used
-  #     github.com/sirupsen/logrus: "logging is allowed only by logutils.Log"
+  depguard:
+    # Rules to apply.
+    # Default: Only allow $gostd in all files.
+    rules:
+      # Name of a rule.
+      main:
+        # List of file globs that will match this list of settings to compare against.
+        # Default: $all
+        files:
+          - "!**/*_a _file.go"
```

@mjtrangoni Does this literally match everything except `**/_a _file.go`? Then, I guess this is basically the same as `$all`.. no?

---

@mumoshu indeed it is like this, but the basic example from upstream use that funny regex. See [here](https://golangci-lint.run/usage/linters/#depguard). Should I change it back to `$all`?

---

Try to use all.

---

Thanks for confirming! Let's use `$all` for now and see how it goes.

```suggestion
          - $all
```